### PR TITLE
[ci] Precompile assets before testing

### PR DESCRIPTION
**Pull request by hallelujah**
We already precompile assets with sprockets, webpacker should follow the same behaviour.

### CODE REVIEW

File: config/webpacker.ci.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    - .sass
+    - .scss
+    - .css
+    - .png
+    - .svg
+    - .gif
+    - .jpeg
+    - .jpg
+
+test:
```
Comment (hallelujah) [inline config/webpacker.ci.yml]: I removed the other environments as this is only needed for CI

File: Dockerfile [hunk 2]
```diff
@@ -47,12 +47,14 @@ VOLUME [ "/opt/system/tmp/cache/", \
          "/opt/system/node_modules", \
          "/opt/system/assets/jspm_packages", \
          "/opt/system/public/assets", \
+         "/opt/system/public/packs-test", \
```
Comment (hallelujah) [inline Dockerfile]: We also need to cache this directory

File: Dockerfile [hunk 3]
```diff
@@ -47,12 +47,14 @@ VOLUME [ "/opt/system/tmp/cache/", \
...
          "/root/.jspm", "/home/ruby/.luarocks" ]
 
 ADD . ./
 ADD config/examples/*.yml config/
 # Needed for Sphinx ODBC
 ADD config/oracle/odbc*.ini /etc/
+COPY config/webpacker.ci.yml config/webpacker.yml
```
Comment (guicassolato) [inline Dockerfile]: Can we have a directory `config/ci` instead?

Comment (guicassolato) [reply Dockerfile]: Maybe in the Dockerfile we can do
```Dockerfile
ADD config/examples/**/*.yml config/
```

but in local keep doing

```shell
cp config/examples/*.yml config/
```

Comment (hallelujah) [reply Dockerfile]: We could but I do not know what to put there?

Comment (guicassolato) [reply Dockerfile]: `condif/examples/ci/webpacker.yml`

Comment (hallelujah) [reply Dockerfile]: Just this file? I do not see the point here

Comment (guicassolato) [reply Dockerfile]: I see now. `webpacker.ci.yml` is not in `config/examples`.

**Pull request merged**